xwiki-platform-core/xwiki-platform-panels/xwiki-platform-panels-ui/src/main/resources/Panels/MyRecentModifications.xml

```diff
@@ -173,15 +173,36 @@
     <property>
       <content>{{velocity}}
 #set ($void = $services.async.useEntity("wiki:$xcontext.database"))
-#set ($query = $services.query.xwql('where doc.author = :author order by doc.date desc'))
+#set ($query = $services.query.hql('select doc.fullName, doc.language from XWikiDocument doc where doc.author = :author order by doc.date desc'))
```

Comment:
You can do a select with xwql too. It's just that it's `Document` instead of `XWikiDocument`.